manifests/params.pp

```diff
@@ -8,9 +8,14 @@
 
   case $::osfamily {
     'Debian': {
+      if $::operatingsystem == ubuntu {
```

Comment:
If you are using a bare word, then it needs to begin with lowercase.

see [https://docs.puppetlabs.com/puppet/4.2/reference/lang_data_string.html#bare-words](https://docs.puppetlabs.com/puppet/4.2/reference/lang_data_string.html#bare-words)

so, can be either format, but `== Ubuntu` would be incorrect